main/main.cpp

```diff
@@ -2132,17 +2126,9 @@ bool Main::start() {
 			ERR_FAIL_COND_V_MSG(da.is_null(), false, "Argument supplied to --doctool must be a valid directory path.");
 		}
 
-#ifndef MODULE_MONO_ENABLED
-		// Hack to define Mono-specific project settings even on non-Mono builds,
-		// so that we don't lose their descriptions and default values in DocData.
-		// Default values should be synced with mono_gd/gd_mono.cpp.
 		GLOBAL_DEF("mono/debugger_agent/port", 23685);
 		GLOBAL_DEF("mono/debugger_agent/wait_for_debugger", false);
 		GLOBAL_DEF("mono/debugger_agent/wait_timeout", 3000);
```

Comment:
Move those up together with the rest, just add a new `#ifdef TOOLS_ENABLED` for them. If they're no longer all defined in the doctool codepath, then they're not reason to define those three here.